visualization/app/codeCharta/ui/customConfigs/downloadCustomConfigsButton/downloadCustomConfigButton.module.ts

```diff
@@ -1,12 +1,12 @@
 import { NgModule } from "@angular/core"
 
-import { DownloadCustomConfigService } from "./downloadCustomConfig.service"
+import { CustomConfigHelperService } from "../customConfigHelper.service"
 import { DownloadCustomConfigsButtonComponent } from "./downloadCustomConfigsButton.component"
 
 @NgModule({
 	declarations: [DownloadCustomConfigsButtonComponent],
 	exports: [DownloadCustomConfigsButtonComponent],
-	providers: [DownloadCustomConfigService],
+	providers: [CustomConfigHelperService],
 	entryComponents: [DownloadCustomConfigsButtonComponent]
```

Comment:
As the parent is now migrated we don't need this entryPoint anymore (probably we should check the others as well)
```suggestion
```